<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="makekeys">
      <property name="text">
       <string>Make keys</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Public</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="publickey">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Private</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="privatekey"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="sign_input">
          <property name="placeholderText">
           <string>Put your hex to firm here</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sign_privatekey">
          <property name="placeholderText">
           <string>Your hex private key</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sign">
          <property name="text">
           <string>Sign</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sign_output">
          <property name="placeholderText">
           <string>Get hex firm here</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="verify_input">
          <property name="placeholderText">
           <string>Put your firmed hex here</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="verify_firm">
          <property name="placeholderText">
           <string>Put hex firm here</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="verify_publickey">
          <property name="placeholderText">
           <string>Your hex public key</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="verify">
          <property name="text">
           <string>Verify</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
